test/ectest.c

```diff
@@ -188,1185 +146,1045 @@ static void group_order_tests(EC_GROUP *group)
+    },
+    /* Curve K-233 (FIPS PUB 186-2, App. 6) */
+    {
+        "NIST curve K-233",
+        "020000000000000000000000000000000000000004000000000000000001",
+        "0",
+        "1",
+        "017232BA853A7E731AF129F22FF4149563A419C26BF50A4C9D6EEFAD6126",
+        "01DB537DECE819B7F70F555A67C427A8CD9BF18AEB9B56E0C11056FAE6A3",
+        0,
+        "008000000000000000000000000000069D5BB915BCD46EFB1AD5F173ABDF",
+        "4", 233
+    },
+    /* Curve B-233 (FIPS PUB 186-2, App. 6) */
+    {
+        "NIST curve B-233",
+        "020000000000000000000000000000000000000004000000000000000001",
+        "000000000000000000000000000000000000000000000000000000000001",
+        "0066647EDE6C332C7F8C0923BB58213B333B20E9CE4281FE115F7D8F90AD",
+        "00FAC9DFCBAC8313BB2139F1BB755FEF65BC391F8B36F8F8EB7371FD558B",
+        "01006A08A41903350678E58528BEBF8A0BEFF867A7CA36716F7E01F81052",
+        1,
+        "01000000000000000000000000000013E974E72F8A6922031D2603CFE0D7",
+        "2", 233
+    },
+    /* Curve K-283 (FIPS PUB 186-2, App. 6) */
+    {
+        "NIST curve K-283",
+        "0800000000000000000000000000000000000000000000000000000000000000"
+            "000010A1",
+        "0",
+        "1",
+                                                                "0503213F"
+        "78CA44883F1A3B8162F188E553CD265F23C1567A16876913B0C2AC2458492836",
+                                                                "01CCDA38"
+        "0F1C9E318D90F95D07E5426FE87E45C0E8184698E45962364E34116177DD2259",
+        0,
+                                                                "01FFFFFF"
+        "FFFFFFFFFFFFFFFFFFFFFFFFFFFFE9AE2ED07577265DFF7F94451E061E163C61",
+        "4", 283
+    },
+    /* Curve B-283 (FIPS PUB 186-2, App. 6) */
+    {
+        "NIST curve B-283",
+                                                                "08000000"
+        "00000000000000000000000000000000000000000000000000000000000010A1",
+                                                                "00000000"
+        "0000000000000000000000000000000000000000000000000000000000000001",
+                                                                "027B680A"
+        "C8B8596DA5A4AF8A19A0303FCA97FD7645309FA2A581485AF6263E313B79A2F5",
+                                                                "05F93925"
+        "8DB7DD90E1934F8C70B0DFEC2EED25B8557EAC9C80E2E198F8CDBECD86B12053",
+                                                                "03676854"
+        "FE24141CB98FE6D4B20D02B4516FF702350EDDB0826779C813F0DF45BE8112F4",
+        1,
+                                                                "03FFFFFF"
+        "FFFFFFFFFFFFFFFFFFFFFFFFFFFFEF90399660FC938A90165B042A7CEFADB307",
+        "2", 283
+    },
+    /* Curve K-409 (FIPS PUB 186-2, App. 6) */
+    {
+        "NIST curve K-409",
+        "0200000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000008000000000000000000001",
```

Comment:
Sorry, missed these.  Fixed now.

My quick reformatting script required manual selection of the string lines.